unit/miniBDD_new.cpp

```diff
@@ -144,11 +150,13 @@ class bdd_propt:public propt
   resultt prop_solve() override
   {
     UNREACHABLE;
+    return {};
```

Comment:
How come these are required?

Reply:
When this is built under CMake, MSVC 2013 complains about missing return statements. I think this file isn't build under the Makefile build, which is why we don't see the error in Appveyor